lib/rules/new-cap.js

```diff
@@ -144,6 +144,22 @@ module.exports = function(context) {
         return false;
     }
 
+    /**
+     * Reports the given message for the given node. The location will be the start of the property or the callee.
+     * @param {ASTNode} node CallExpression or NewExpression node.
+     * @param {string} message The message to report.
+     * @returns {void}
+     */
+    function report(node, message) {
+        var callee = node.callee;
+
+        if (callee.type === "MemberExpression") {
+            callee = callee.property;
+        }
+
+        context.report(node, callee.loc.start, message);
```

Comment:
It's fine either way - no one sees the node type except in tests. :) Since I'd like to get a new release out today, I'd say leave as-is.